webapp/packages/core-authentication/src/TeamsResource.ts

```diff
@@ -16,33 +16,50 @@ import {
   ResourceKeyUtils,
   AdminTeamInfoFragment,
   AdminConnectionGrantInfo,
-  CachedMapAllKey
+  CachedMapAllKey,
+  GetTeamsListQueryVariables
 } from '@cloudbeaver/core-sdk';
 import { isArraysEqual } from '@cloudbeaver/core-utils';
 
 const NEW_TEAM_SYMBOL = Symbol('new-team');
 
 export type TeamInfo = AdminTeamInfoFragment;
+
+type TeamResourceIncludes = Omit<GetTeamsListQueryVariables, 'teamId'>;
 type NewTeam = TeamInfo & { [NEW_TEAM_SYMBOL]: boolean; timestamp: number };
 
 @injectable()
-export class TeamsResource extends CachedMapResource<string, TeamInfo> {
+export class TeamsResource extends CachedMapResource<string, TeamInfo, TeamResourceIncludes> {
   constructor(private readonly graphQLService: GraphQLService) {
     super();
   }
 
   async loadAll(): Promise<Map<string, TeamInfo>> {
+    this.resetIncludes();
```

Comment:
reset includes in loading function like in ConnectionsInfoResource:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/811729/199050706-e7703c4a-681d-4148-b46b-7c5c3a15ba3f.png">